EmbeddedPkg/Library/FdtLib/fdt_overlay.c

```diff
@@ -385,6 +385,10 @@ overlay_update_local_node_references (
                                       );
     int  tree_child;
 
+    if (fixup_child_name == NULL) {

```

Comment:
Sorry you've spent time on this, but this haphazard libfdt-integration is going away very soon, and I don't really want to make it look like someone cares about it. Could you drop the changes here please?
All code should be using MdePkg/Library/BaseFdtLib instead.

Reply:
Thanks for letting me know I have reverted the changes.